src/utils/api.js

```diff
@@ -2,6 +2,20 @@
 import { getToken } from './tokenStorage';
 import { serverBaseUrl } from '../config';
 
+export type ApiError = {
+  message: string,
+  response: {
+    status: number,
+    statusText: string,
+  },
+};
+
+type BaseApiResponse = {
+  error?: ApiError,
+};
+
+type DataResponse<D> = Promise<BaseApiResponse & { data?: D }>;
```

Comment:
Note that we can use the `DataResponse` type to tell Flow what sort of data to expect from an API call.
This is used in f.x. `getPoliticians`